spec/domain/person/filter/role_spec.rb

```diff
@@ -337,9 +337,9 @@ def filter(attrs)
           expect(filter(start_at: now).entries).to be_empty
         end
 
-        it 'finds role deleted within range' do
+        it 'does not find role deleted within range' do
```

Comment:
Ist das wirklich korrekt? Habe ich nichts kaputt gemacht mit meinen Bugfixes zu zukünftigem Löschdatum?

Reply:
Du hast recht merci.
Tatsächlich sind die Änderungen im filter/role.rb so nicht ganz richtig. Abhängig vom args[:kind] werden bereits beim `roles_join` die gelöschten anhand `Time.now.utc` herausgefiltert. Ab dem Moment wenn aber nach aktiven in einem vergangenen Zeitraum gefiltert wird, findet es dann die Rollen die damals vielleicht noch aktiv waren gar nicht mehr.

Ich hab jetzt die Änderungen zurückgesetzt und im Commit nachgeschaut. Der Spec in deleted_people_spec.rb läuft immernoch durch.. Vielleicht fehlt da aber noch eine Spec und das muss manuell genauer behandelt werden?
https://github.com/hitobito/hitobito/pull/1963/commits/597f4dedce87c5af9f9026592ceb2abc6a2aa73f